src/bezier_spline.inc

```diff
@@ -224,26 +230,97 @@ template <std::size_t parametric_dimension, typename PhysicalPointType,
           typename ScalarType>
 constexpr BezierSpline<parametric_dimension, PhysicalPointType, ScalarType>
 BezierSpline<parametric_dimension, PhysicalPointType, ScalarType>::
-    DerivativeWRTParametricDimension(const IndexingType par_dim) const {
-  // Initialize return value and precompute auxiliary values
+    DerivativeWRTParametricDimension(
```

Comment:
would you like to write the equation you actually implement? you can also do this in splinepy ;)